build/tools/SharedCodeGenerator/Outputs/CSharpConstants.cs.tpl

```diff
@@ -2,7 +2,7 @@
 
 namespace {{ Namespace }}
 {
-    public static class {{ Name }}
+    {{ Scope }} static class {{ Name }}
```

Comment:
For example in case of detector we want to limit the types which are exposed as public